airflow/www/views.py

```diff
@@ -1205,6 +1205,13 @@ def tree(self):
         dag_id = request.args.get('dag_id')
         blur = conf.getboolean('webserver', 'demo_mode')
         dag = dagbag.get_dag(dag_id)
+        if not dag:
+            flash(
+                "DAG [{}] doesn't seem to exist"
+                " at the moment".format(dag_id),
+                "error")
+            return redirect('/admin/')
```

Comment:
What would you prefer? I was being consistent with pre-existing behavior in https://github.com/vijaysbhat/incubator-airflow/blob/312de3a24c0753c88ad9734495ac572aac03f7bd/airflow/www/views.py#L804-L809, https://github.com/vijaysbhat/incubator-airflow/blob/312de3a24c0753c88ad9734495ac572aac03f7bd/airflow/www/views.py#L1341-L1343 and https://github.com/vijaysbhat/incubator-airflow/blob/312de3a24c0753c88ad9734495ac572aac03f7bd/airflow/www/views.py#L879-L884.